tests/core/trace/ctfTracerTest.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019 fotiss GmbH
+ * Copyright (c) 2024 Jose Cabral
```

Comment:
I am not sure if changing the copyright here from Fortiss GmbH to exclusively Jose Cabral is valid here, as much of the original code is still present.
Even if you wrote the code originally, it seems you transferred the copyright to Fortiss back then.

@azoitl

Reply:
This was a correction from a commit from a couple of months ago actually. The original file is not from 2019 and not done at fortiss, but by me this year